aspnetcore/host-and-deploy/docker/building-net-docker-images.md

```diff
@@ -170,12 +178,12 @@ In some scenarios, you might want to deploy an app to a container by copying to
 
 * Browse to `http://localhost:5000` to see the home page.
 
-To use the manually published application within a Docker container, create a new Dockerfile and use the `docker build .` command to build the container.
+To use the manually published app within a Docker container, create a new Dockerfile and use the `docker build .` command to build the container.
```

Comment:
Italicize file names

```suggestion
To use the manually published app within a Docker container, create a new *Dockerfile* and use the `docker build .` command to build the container.
```

Reply:
@scottaddie, (Just got back from several weeks of vacation, hence the long time before a reply),
I totally agree and thanks for catching that I missed the reference to the file name.    I do have a question though:  In this case we refer to a file name *Dockerfile*, but it is an odd one where also refer to it as a type of file with a specific function, such as "Look for the Dockerfile in the current folder".   In the latter case, do we also use italics or any other special format?  Thanks in advance.